modules/ossm-automatic-sidecar-injection.adoc

```diff
@@ -6,20 +6,24 @@
 [id="ossm-automatic-sidecar-injection_{context}"]
 = Enabling automatic sidecar injection
 
-When deploying an application into the {ProductName} you must opt in to injection by specifying the `sidecar.istio.io/inject` annotation with a value of `"true"`. Opting in ensures that the sidecar injection does not interfere with other OpenShift features such as builder pods used by numerous frameworks within the OpenShift ecosystem.
+When deploying an application, you must opt-in to injection by setting the `sidecar.istio.io/inject` annotation to `"true"`. Opting in ensures that the sidecar injection does not interfere with other OpenShift features such as builder pods used by numerous frameworks within the OpenShift ecosystem.
 
 .Prerequisites
 
 * Identify the deployments for which you want to enable automatic sidecar injection.
-* Locate the application's YAML configuration file.
 
 .Procedure
 
-. Open the application's configuration YAML file in an editor.
+. Open the application's deployment configuration YAML file in an editor. To find a deployment use the `oc get` command. For example, for an app called `sleep` in the `sleep` namespace, use the following command to see the resource in YAML format.
++
+[source,yaml]
+----
+oc get deployment sleep -o yaml
+----
 
-. Add `sidecar.istio.io/inject` to the configuration YAML with a value of `"true"` as illustrated here:
+. Add `sidecar.istio.io/inject` to the configuration YAML with a value of `"true"` in the `spec.template.metadata.annotations.sidecar.istio/inject` field. See the following example for an app called `sleep`.
 +
-.Sleep test application example
+.Sleep test application example sleep.yaml
 [source,yaml]
 ----
 apiVersion: apps/v1
```

Comment:
The sleep yaml sample image: tutum/curl has been deleted from upstream dockerhub. You may replace that with
```
image: curlimages/curl
```
and replace the command with
```
command: ["/bin/sleep","3650d"]
```